server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequestBuilder.java

```diff
@@ -162,6 +162,17 @@ public RestoreSnapshotRequestBuilder setIncludeAliases(boolean restoreAliases) {
         return this;
     }
 
+    /**
+     * If set to true the logging should be at a lower level
+     *
+     * @param silent true if logging should be at a lower level
+     * @return this builder
+     */
+    public RestoreSnapshotRequestBuilder setSilent(boolean silent) {
```

Comment:
I don't think this is used, so it can be dropped. These request builders are a bit of a legacy thing for use in clients that no longer exist. Mostly it's fine just to create the request directly.